code/game/objects/items/melee/hf.dm

```diff
@@ -0,0 +1,152 @@
+	desc = "RULES OF NATURE"
+	icon_state = "hfblade"
+	worn_icon_state = "hfblade"
+	icon = 'icons/obj/hfblade.dmi'
+	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	flags_1 = CONDUCT_1
+	obj_flags = UNIQUE_RENAME
+	force = 0.001
+	throwforce = 20
+	w_class = WEIGHT_CLASS_BULKY
+	block_chance = 50
+	armour_penetration = 75
+	sharpness = SHARP_EDGED
+	attack_verb_simple = list("chopped", "sliced", "cut", "zandatsu'd")
+	attack_verb_continuous = list("chopped", "sliced", "cut", "zandatsu'd")
+	hitsound = 'sound/weapons/rapierhit.ogg'
+	custom_materials = list(/datum/material/iron = 10000)
+	light_color = "#40ceff"
+	var/rules_of_nature = TRUE // Turn this off to break the rules, and watch the horrors of no proximity flags unfold.
+	var/brazil = FALSE
+	var/brightness = 5
+
+/obj/item/melee/hfblade/Initialize()
+	. = ..()
+	AddComponent(/datum/component/butchering, 30, 95, 5)
+	set_light(brightness)
+	START_PROCESSING(SSobj, src)
+
+/obj/item/melee/hfblade/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+
+/obj/item/melee/hfblade/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(attack_type == PROJECTILE_ATTACK)
+		final_block_chance = 0
+	if(attack_type == MELEE_ATTACK) // It really is not a good idea to get close to this thing.
+		final_block_chance = 100
+		var/hideandseeklogic = rand(1,5) // YES, ADD MORE RNG TO COMBAT.
+		if(ishuman(hitby.loc))
+			var/mob/living/carbon/human/attacker = hitby.loc
+			if(hideandseeklogic == 1 || HAS_TRAIT(attacker, TRAIT_CLUMSY))
+				attacker.visible_message("<span class = 'warning'>[attacker] attempts to attack [owner] and accidentally slices their arm off! What an idiot!</span>")
+				var/which_hand = BODY_ZONE_L_ARM
+				if(!(attacker.active_hand_index % 2))
+					which_hand = BODY_ZONE_R_ARM
+				var/obj/item/bodypart/disarm_arm = attacker.get_bodypart(which_hand)
+				disarm_arm.dismember()
+				playsound(attacker,pick('sound/misc/desecration-01.ogg','sound/misc/desecration-02.ogg','sound/misc/desecration-01.ogg') ,50, 1, -1)
+	return ..()
+
+/obj/item/melee/hfblade/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	if(!proximity_flag && rules_of_nature)
+		return
+	var/list/parts = list()
+	if(iscarbon(target))
+		var/mob/living/carbon/C = target
+		for(var/X in C.bodyparts)
+			var/obj/item/bodypart/BP = X
+			if(BP.body_part != HEAD && BP.body_part != CHEST)
+				if(BP.dismemberable)
+					parts += BP
+	if(parts)
+		if(!parts.len)
+			return FALSE
+		var/mob/living/carbon/C = target
+		var/obj/item/bodypart/BP = pick(parts)
+		BP.dismember()
+		playsound(C,pick('sound/misc/desecration-01.ogg','sound/misc/desecration-02.ogg','sound/misc/desecration-01.ogg') ,50, 1, -1)
+		C.adjustBruteLoss(-15) //dismembering a limb deals 15 brute, this heals you instantly after that
```

Comment:
This may have been changed with wounds

Reply:
yeah hold on lemme test if that's true
somehow
uh

Reply:
ok I understand how to test now